functionnal_tests/cypress/integration/app_workspace_advanced/read_mode_only_spec.js

```diff
@@ -1,84 +1,107 @@
-import { PAGES } from '../../support/urls_commands'
-import { SELECTORS } from '../../support/generic_selector_commands'
+import { PAGES as p } from '../../support/urls_commands'
+import { SELECTORS as s } from '../../support/generic_selector_commands'
 
 describe('Space settings in reader mode', () => {
   let workspaceId
   const flashMessageClass = '.flashmessage__container__content__text__paragraph'
   const flashMessageText = 'Your tag has been created'
 
-  before(function  () {
-    this.skip() // MB - 2021-10-11 - unstable test, see issue : https://github.com/tracim/tracim/issues/4995
-
+  before(() => {
     cy.resetDB()
     cy.setupBaseDB()
+
     cy.loginAs('administrators')
+
     cy.fixture('baseWorkspace').as('workspace').then(workspace => {
       workspaceId = workspace.workspace_id
-      cy.visitPage({ pageName: PAGES.DASHBOARD, params: { workspaceId } })
-      cy.contains('.userstatus__role__text', 'Space manager')
-      cy.getTag({ selectorName: SELECTORS.WORKSPACE_DASHBOARD })
+
+      cy.visitPage({ pageName: p.DASHBOARD, params: { workspaceId } })
+
+      // NOTE - MP - 05-11-2021 - If the page isn't loaded after 30s
+      // there is a problem somewhere
+      cy.get('.userstatus__role__text', { timeout: 30000 })
+        .contains('Space manager')
+
+      cy.getTag({ selectorName: s.WORKSPACE_DASHBOARD })
         .find('.dashboard__workspace__detail__buttons .iconbutton')
         .click()
 
+      // NOTE - MP - 05-11-2021 - Add a tag to test the tag manipulation
       cy.get('[data-cy=popin_right_part_tag').click()
       cy.get('[data-cy=add_tag]').type('Tag')
       cy.get('[data-cy=validate_tag]').click()
-      cy.contains(flashMessageClass, flashMessageText)
 
+      // NOTE - MP - 05-11-2021 - Change visibility
       cy.get('[data-cy=popin_right_part_members_list]').click()
-      cy.get('.workspace_advanced__userlist__list__item__role').last().click()
-      cy.get('.workspace_advanced__userlist__list__item__role .dropdownMenuItem .fa-gavel')
-        .last().click()
-      cy.get('.workspace_advanced__userlist__list__item__role').first().click()
-      cy.get('.workspace_advanced__userlist__list__item__role .dropdownMenuItem .fa-eye')
+      // NOTE - MP - 05-11-2021 - 2 is the id of John Doe
+      cy.get('[data-cy=workspace_advanced__member-2_role]')
+        .find('button')
+        .first().click()
+      cy.get('[data-cy=workspace_advanced__member-2_role]')
+        .find('[data-cy=dropdownMenu_items]')
+        .find('>span')
+        // FIXEME - MP - 05-11-2021 - Add data-cy into theses button
```

Comment:
```suggestion
        // FIXME - MP - 05-11-2021 - Add data-cy into these buttons
```
Is there an issue to fix this problem?

Reply:
It looks like this more of a note than a fixme

Reply:
@Haskaris do you agree? If those are FIXMEs, could you link them to issues? I will leave it like that for now.

Reply:
Actually it's a mistake, I wrote these ``FIXMEs`` before doing it and forgetting to delete these comments.
Just in case, if they were still relevant (changes not made), I think it should be ``TODO`` instead of ``FIXEME`` so we can get a good practice of doing Cypress tests (see [Best practices](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)).